src/frontend/src/views/Data/components/TreeTableNameCell/components/PhyloTreeStatusTag/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { TREE_STATUS } from "src/common/constants/types";
+import { StyledChip } from "./style";
+
+// TODO (mlila): this should actually be exported from sds
+export type CHIP_STATUS =
+  | "success"
+  | "error"
+  | "warning"
+  | "info"
+  | "pending"
+  | "beta";
```

Comment:
what is `beta` for?

Reply:
beta is an SDS color set not currently used in Aspen. It would be used for visual elements that highlight something as a beta feature for users.

Reply:
oh ok, not sure if we need this here then? But also not an issue